relayer/src/message_relayer/mod.rs

```diff
@@ -114,7 +114,7 @@ async fn submit_message(
 
     // TODO: Fully decode
     let nonce_bytes = &message.nonce_le[..16];
-    let nonce = u128::from_le_bytes(nonce_bytes.try_into()?);
```

Comment:
Take all the data from `message.nonce_le`

Reply:
Also there's another usage of `provide_content_message` in `message_processor.rs`